---
layout: page
title: tools and code

tools:
- name        : Antgenomes.org
  img         : antgenomes.png
  url         : http://antgenomes.org
  about       : Initially known as Fourmidable. Community database centralizing genomic and transcriptomic sequence information for ants. 
  paper : /publications/wurm2009antDatabase.pdf
  status      : Used by hundreds of researchers. 
  status_icon : fa-certificate

- name        : SequenceServer
  img         : sequenceserver.png
  url         : http://sequenceserver.com
  code        : http://github.com/yannickwurm/sequenceserver
  about       : "BLAST is the most commonly used bioinformatics tool. But setting it up for private data and using it is counter-intuitive. We're developing SequenceServer to make BLAST easy to use."
  status      : Deployed in dozens of institutions.
  status_icon : fa-certificate

- name        : "Codename Afra"
  img         : afra.png
  url         : http://afra.sbcs.qmul.ac.uk
  code        : http://github.com/yeban/afra
  about       : <a href="http://bbsrc.ac.uk">BBSRC</a>-funded tool for crowd-sourcing curation of gene predictions.
  status      : Alpha. Work in progress.
  status_icon : fa-warning

- name        : GeneValidator
  img         : genevalidator.png
  code        : https://github.com/monicadragan/GeneValidator
  about       : Tool to identify problematic gene predictions based on comparison with gene predictions in public databases. Funded by <a href="http://informatics.nescent.org/wiki/Phyloinformatics_Summer_of_Code_2013">NESCent Google Summer of Code 2013</a>.
  status      : Alpha. Work in progress.
  status_icon : fa-warning

- name        : Transcript-genome consistency network
  img         : tgnet.jpg
  code        : https://github.com/ksanao/TGNet
  about       : <a href="https://github.com/ksanao/TGNet">TGNet</a> is a <a href="http://www.cytoscape.org">Cytoscape</a>-based tool for visualization and quality assessment of de novo genome assemblies. Specifically it facilitates rapid detection of inconsistencies between a genome assembly and an independently derived transcriptome assembly. Developed by <a href="http://scholar.google.co.uk/citations?user=aovYPpkAAAAJ&hl=en">Oksana Riba-Grognuz</a>.
  paper       : http://bioinformatics.oxfordjournals.org/content/27/24/3425
  status_icon : fa-certificate
  status      : Used for <a href="http://www.pnas.org/content/early/2011/01/24/1009690108">fire ant genome</a> project. Published in <a href="http://bioinformatics.oxfordjournals.org/lookup/doi/10.1093/bioinformatics/btr569">Bioinformatics</a>. 

- name        : NeuroPeptide Search
  img         : NpSearch.png
  code        : https://github.com/IsmailM/NeuroPeptideSearch
  about       : Tool to identify all potential neuropeptide precursors in a dataset. NeuroPeptide Search utilises HMM models and common features of neuropeptides to identify neuropeptide homologs and potential novel neuropeptides. Neuropeptide Search has already been successfully used to analyse the transcriptomes of ten different species, resulting in the discovery in numerous novel neuropeptides.  
  status      : Beta. Functioning beta version available.
  status_icon : fa-warning


repositories:
 - name: Anurag Priyam
   repository: https://github.com/yeban
 - name: Yannick Wurm
   repository: https://github.com/yannickwurm
 - name: Monica Dragan
   repository: https://github.com/monicadragan
 - name: Bruno Vieira
   repository: https://github.com/bmpvieira
 - name: Ismail Moghul
   repository: https://github.com/IsmailM

---

<div class="row">
  <h3>Major tools we develop include:</h3>
  {% for tool in page.tools %}
  <div class="span5">
    <div class="caption">
      <a href="{{ tool.url }}"><img src="/img/{{ tool.img }}" alt="{{ tool.name }}" align="right"
				    style="margin-right:10px; height:180px;"></a>
      <h4>{{ tool.name }}</h4>
	<div class="row">
	  <ul class="fa-ul">
		<li><i class="fa-li fa {{tool.status_icon}}"></i>{{tool.status}}</li>
		<li><i class="fa-li fa fa-info-circle"></i>{{ tool.about }}</li>
		{% if tool.code %}  <li><i class="fa-li fa fa-code"></i><a href="{{tool.code}}">{{ tool.code }}</a></li>{%endif%}
		{% if tool.url %}   <li><i class="fa-li fa fa-external-link-square"></i><a href="{{tool.url}}">{{ tool.url }}</a></li>{%endif%}
  		{% if tool.paper %} <li><i class="fa-li fa fa-file-text"></i><a href="{{tool.paper}}">Publication</a></li>{%endif%}

	  </ul>
	  </div>
	  </div>
  </div>
  {% cycle '', '</div><div class="row">' %} 
  {% endfor %}
</div>

<h3>Approaches</h3>

<p>We believe in <a
href="http://www.nature.com/nature/focus/reproducibility/">Reproducible
Research</a> &amp; <a href="http://software.ac.uk">Software
Sustainability</a>. When possible we respect style guides and use unit
testing &amp; continuous integration. We develop mostly in:</p>
<ul>
  <li><a href="http://bioruby.org">ruby/bioruby</a> (when making
  pipelines, text &amp; backends;</li>
  <li><a href="http://rproject.org">R</a> for number crunching</li>
  <li>javascript/html/css for client-side stuff</li>
  <li>But also use bash/python/perl when convenient or necessary...</li>
</ul>
<p>But we love quick and dirty hacks too <i class="fa fa-smile-o"></i>.</p>

<h3>Source code repositories</h3>
<p>Several <a href="/team.html">team members</a> have open source
  code repositories which include all sorts of useful scripts: </p>
<ul class="fa-ul">
	{%for member in page.repositories %}
	<li><i class="fa-li fa fa-github"></i><a
	href="{{member.repository}}">{{ member.name }}'s GitHub</a></li>
	{%endfor %}
</ul>

  


